apps/prairielearn/src/lib/submissions-upload.ts

```diff
@@ -140,29 +165,85 @@ export async function uploadSubmissions(
     });
     const csvParser = createCsvParser(csvStream, {
       lowercaseHeader: false,
-      maxRecordSize: 1 << 20, // 1MB (should be plenty for a single line)
+      maxRecordSize: 1 << 24, // 16MB (should be plenty for a single line)
     });
+
+    // Detect if this is a group work CSV by checking the first row
+    let isGroupWork: boolean | null = null;
+
     for await (const { info, record } of csvParser) {
       job.verbose(`Processing CSV line ${info.lines}: ${JSON.stringify(record)}`);
```

Comment:
@reteps this was probably the reason you were unable to load the server job logs. We should probably log this more intelligently, e.g. only log the UID/group name by default, and log the row truncated to N (200?) characters if there's an error, or something like that.